compiler/fir/raw-fir/raw-fir.common/src/org/jetbrains/kotlin/fir/builder/AbstractRawFirBuilder.kt

```diff
@@ -91,21 +91,27 @@ abstract class AbstractRawFirBuilder<T : Any>(val baseSession: FirSession, val c
         name: Name,
         isExpect: Boolean,
         forceLocalContext: Boolean = false,
+        containedWithinInlineFunction: Boolean? = null,
         l: () -> T,
     ): T = when {
         forceLocalContext -> withForcedLocalContext {
-            withChildClassNameRegardlessLocalContext(name, isExpect, l)
+            withChildClassNameRegardlessLocalContext(name, isExpect, containedWithinInlineFunction, l)
         }
         else -> {
-            withChildClassNameRegardlessLocalContext(name, isExpect, l)
+            withChildClassNameRegardlessLocalContext(name, isExpect, containedWithinInlineFunction, l)
         }
     }
 
     inline fun <T> withChildClassNameRegardlessLocalContext(
         name: Name,
         isExpect: Boolean,
+        containedWithinInlineFunction: Boolean? = null,
         l: () -> T,
```

Comment:
This function is called only from `withChildClassName`, so the parameter could be removed from there too.